### PR TITLE
Add custom network and Import custom token

### PR DESCRIPTION
**Pull request by chrisleewilcox**
https://app.zenhub.com/workspaces/mobile-automation-board-633766327776c3e8970e0a94/issues/gh/metamask/mobile-planning/544

**Description**
Add custom network and import custom token has been added to the automation.

**Screenshots/Recordings**

_If applicable, add screenshots and/or recordings to visualize the before and after of your change_

**Issue**

Progresses #???

**Checklist**

* [ ] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

### CODE REVIEW

File: wdio/features/screen-objects/AddCustomImportTokensScreen.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        return Selectors.getElementByPlatform(TOKEN_ADDRESS_SYMBOL_ID);
+     }
+  
+    async typeCustomTokenAddress(text) {
+        await Gestures.typeText(this.customTokenAddressField, text);
+    }
+
+    async tapImportButton() {
+        await driver.pause(3000);
+        await Gestures.tapTextByXpath('IMPORT');
```
Comment (cortisiko) [inline wdio/features/screen-objects/AddCustomImportTokensScreen.js:27]: tap by ID instead of xpath

**Review by cortisiko — CHANGES_REQUESTED** [review]
small tings.

File: wdio/features/screen-objects/AddCustomImportTokensScreen.js (hunk 1 above)
Comment (chrisleewilcox) [reply wdio/features/screen-objects/AddCustomImportTokensScreen.js:27]: Updated to use ID, works.

File: wdio/features/screen-objects/AddCustomImportTokensScreen.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        await driver.pause(3000);
+        await Gestures.tapTextByXpath('IMPORT');
+    }
+
+    async tapTokenSymbolField(){
+        await Gestures.tap(this.symbolField);
+    }
+
+    async typeCustomTokenSymbol(symbol){
```
[the remark this answers is not in the document]
Comment (chrisleewilcox) [reply wdio/features/screen-objects/AddCustomImportTokensScreen.js:36]: Using Gestures.tap now, works.

**Review by cortisiko — APPROVED** [review]
🚀 🚀 🚀 🚀

**Pull request merged**